### PR TITLE
Olwa update

### PR DESCRIPTION
**Pull request by oliver-walker-hms**

### CODE REVIEW

**Comment by oliver-walker-hms** [comment]
with all the library updates from last pr a new release would be nice

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Flexy OSIsoft Changelog
 
+## v2.2
+### Major Changes
++ Updated Solutions Center libraries
```
Comment (TomKimsey) [inline docs/CHANGELOG.md:5]:
```suggestion
+ Updated Solution Center libraries

```

Comment (alexjhawk) [inline docs/CHANGELOG.md:5]:
```suggestion
+ Updated solution center libraries

```

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Flexy OSIsoft Changelog
 
+## v2.2
+### Major Changes
++ Updated solution center libraries
```
Comment (TomKimsey) [inline docs/CHANGELOG.md:5]:
```suggestion
+ Updated Solution Center libraries
```

Comment (oliver-walker-hms) [reply docs/CHANGELOG.md:5]: oh

Comment (TomKimsey) [reply docs/CHANGELOG.md]: Solution Center is a proper noun

**Pull request merged**